code/modules/antagonists/changeling/powers/regenerate.dm

```diff
@@ -1,6 +1,6 @@
 /datum/action/changeling/regenerate
 	name = "Regenerate"
-	desc = "Allows us to regrow and restore missing external limbs and vital internal organs, as well as removing shrapnel and restoring blood volume. Costs 10 chemicals."
+	desc = "Allows us to regrow and restore missing external limbs and vital internal organs, as well as removing shrapnel, healing all wounds, and restoring blood volume. Costs 10 chemicals."
```

Comment:
you need to specify that this is in reference to "wounds" as in the SS13 game mechanic, not "wounds" as in the synonym for damage, or else new changeling players are gonna think that this is fleshmend lite.

Reply:
Oh please

Reply:
sjhut he fuck up[

Reply:
It's a reasonable concern, but I can't think of a way to word it which both specifies that information and isn't super clumsy.

Reply:
Honestly I don't see a reason that healing wounds shouldn't be relegated to fleshmend in the first place, but eh.

Reply:
ok, how about "healing major wounds" instead?

Reply:
yes, healing major wounds works better